package.json

```diff
@@ -22,21 +22,22 @@
         "tsc:build": "tsc --noemit && npm run tsc:build:site"
     },
     "devDependencies": {
-        "@tsconfig/node18": "^18.2.2",
+        "@tsconfig/node20": "^20.1.2",
+        "@types/node": "^20.10.5",
         "@typescript-eslint/eslint-plugin": "^6.8.0",
         "@typescript-eslint/parser": "^6.8.0",
         "@vitest/coverage-c8": "^0.29.2",
-        "aws-cdk": "2.95.1",
-        "aws-cdk-lib": "2.95.1",
-        "constructs": "10.2.69",
+        "aws-cdk": "^2.110.1",
+        "aws-cdk-lib": "^2.110.1",
+        "constructs": "^10.3.0",
         "eslint": "^8.34.0",
         "eslint-config-prettier": "^8.6.0",
         "eslint-import-resolver-typescript": "^3.5.3",
         "eslint-plugin-import": "^2.27.5",
         "eslint-plugin-prettier": "^4.2.1",
         "husky": "^8.0.0",
         "prettier": "^2.8.4",
-        "sst": "2.30.2",
+        "sst": "^2.38.7",
```

Comment:
Better to keep the versions of sst, cdk and constructs pinned so it doesn't end up making infrastructure changes without us knowing beforehand

Reply:
done